frontend/package.json

```diff
@@ -3,6 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@apollo/client": "^3.3.16",
```

Comment:
I don't think we need apollo so this can be removed. Remember to run yarn start after removing it and test to see if the website runs using docker-compose up afterwards!